### DISCUSSION TITLE
'Linked Section' Field = Allow for specific entry type selection within a Section.

### DISCUSSION THREAD

**Discussion by Steve-JG** [page-8/craft-manytomany#49] [Ideas]
Would it be possible to target a specific _**Entry Type**_ within a section - with the _'Linked Section'_ field?

### Usage Scenario -
Say I have a two-level **_'Homes'_** structure with two entry types:

- **House Type -**  used for the parent/Level 1 pages which categorises my homes.
- **Listing -** used for the child/level 2 pages where I create my homes.

Now I have an '**_Upgrades_**' associated field, where I can choose to apply available upgrades for certain house types. e.g. a swimming pool or fireplace won't be available for _Townhouses_ but they will be available for _New Homes_.

Currently, it would be possible for me to specify upgrades for the individual homes _(123 Fake Street etc..)_, however I don't want this selection to be possible for the user, only the parent House Type.

![image](https://github.com/user-attachments/assets/c72fc5f3-415b-48ba-8404-53e7526b07ac)

**Comment by engram-design** [comment]
Not possible at the moment, but it _should_ be something that we can implement without too much trouble at face value. I'll look into it!

**Reply by Steve-JG** [reply]
Thank you mate :) Appreciate the consideration.
